front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -48,9 +48,20 @@ const ConsistCardTitle = ({
 export type StdcmConsistProps = {
   isDebugMode: boolean;
   disabled?: boolean;
+  consistErrors?: ConsistErrors;
+  setConsistErrors?: (errors: ConsistErrors) => void;
 };
 
-const StdcmConsist = ({ isDebugMode, disabled = false }: StdcmConsistProps) => {
+const StdcmConsist = ({
+  isDebugMode,
+  disabled = false,
+  consistErrors = {
+    totalMass: { display: false },
+    totalLength: { display: false },
+    maxSpeed: { display: false },
+  },
+  setConsistErrors = () => {},
```

Comment:
Their types in props should be changed, they are always defined
```suggestion
  consistErrors,
  setConsistErrors,
```

Reply:
done